packages/gitmoji-changelog-cli/src/index.js

```diff
@@ -1,9 +1,9 @@
 #! /usr/bin/env node
-import fs from 'fs'
-import { changelog } from '@gitmoji-changelog/core'
-import { convert } from '@gitmoji-changelog/markdown'
+const fs = require('fs')
```

Comment:
you can use esm if you want:
`node -r esm .`

https://www.npmjs.com/package/esm

Reply:
thx ;) I'm going to check that